src/components/AutoCompleteText/getSuggestions.js

```diff
@@ -1,19 +1,11 @@
 export default suggestions => value => {
   const inputValue = value.trim().toLowerCase();
   const inputLength = inputValue.length;
-  let count = 0;
 
   return inputLength === 0
-    ? []
-    : suggestions.filter(suggestion => {
-        const keep =
-          count < 5 &&
-          suggestion.slice(0, inputLength).toLowerCase() === inputValue;
-
-        if (keep) {
-          count += 1;
-        }
-
-        return keep;
-      });
+    ? suggestions
+    : suggestions.filter(
+        suggestion =>
+          suggestion.slice(0, inputLength).toLowerCase() === inputValue
+      );
```

Comment:
Can we sort alphabetically here, while we're at it? Or is that better done in the components that use this?

Reply:
I think it's best to do it in the component that uses it so that we can use `useMemo` to memoize the list and avoid having to sort the list on each render. I'll file an issue for that.